manifests/kerberos.pp

```diff
@@ -78,7 +89,19 @@
     }
   }
 
-  if ( $createhostkeytab and $createhostuser )
+  if (lookup(profile_secrets::enable))
+  {
+    $vault_uri = profile_secrets::lookup_uri($vaultsecretdir)
+    $vault_auth = lookup(profile_secrets::vault_authmethod)
+    $vault_kv_version = lookup(profile_secrets::vault_kv_version)
+    $hostkeytabbase64 = Sensitive(vault_key($vault_uri,$vault_auth,$vaultkeytabkey,$vault_kv_version))
```

Comment:
I think we want to avoid calling this the `hostkeytab`... as that is the keytab file that will later be created. 
Can you continue to use the `createhostkeytab` naming so that `$hostketabbase64` is switched to be `$createhostkeytabbase64`?

And also updated in the few lines below this?

Reply:
This has been changed